src/views/landing.vue

```diff
@@ -96,7 +96,7 @@ export default class Landing extends Vue {
 }
 
 #upper-row {
-  background: url('../assets/images/analyze-name-bg.jpg') no-repeat bottom;
+  background: url('~@/assets/images/analyze-name-bg.jpg') no-repeat bottom;
```

Comment:
This is a weird-looking path but I tried without `~` and it didn't work; this does. So here it is.

Reply:
Interesting!